database/metric_correlations.c

```diff
@@ -4,7 +4,8 @@
 #include "KolmogorovSmirnovDist.h"
 
 #define MAX_POINTS 10000
-int enable_metric_correlations = CONFIG_BOOLEAN_YES;
+int enable_metric_correlations = CONFIG_BOOLEAN_NO;
```

Comment:
How is it enabled?

Reply:
It is part of the previous PR, there is code to read it from the configuration -> https://github.com/netdata/netdata/blob/108bfb7918a86255bf79119f9b8eee604943ae76/daemon/main.c#L558